manual/manual/refman/lex.etex

```diff
@@ -171,6 +176,17 @@ To allow splitting long string literals across lines, the sequence
 followed by any number of spaces and horizontal tabulations at the
 beginning of the next line) is ignored inside string literals.
 
+Quoted string literals provide an alternative lexical syntax for
+string literals. They are useful to represent strings of arbitrary content
+without escaping. Quoted strings are delimited by a matching pair
+of @'{' quoted-string-id '|'@ and @'|' quoted-string-id '}'@ with
+the same @quoted-string-id@ on both sides. Quoted strings do not interpret
+any character in a special way but requires that the
+sequence @'|' quoted-string-id '}'@ does not occur in the string itself.
+The identifier @quoted-string-id@ is a (possibly empty) sequence of
+lowercase letters and underscores that can be freely chosen to avoid
+such issue.
+
```

Comment:
Good, but I'd give two examples, like `{|hello|}` and `{foo|{|hi|}there|foo}`.

Reply:
Probably (even if examples go a bit against the tone of the chapter).